plugin/storage/cassandra/schema.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+CQLSH_HOST=${CQLSH_HOST:-"cassandra"}
+CASSANDRA_WAIT_TIMEOUT=${CASSANDRA_WAIT_TIMEOUT:-"60"}
+DATACENTER=${DATACENTER:-"openshift"}
+KEYSPACE=${KEYSPACE:-"jaeger_v1_${DATACENTER}"}
+
+total_wait=0
+while true
+do
+  ping -c 1 ${CQLSH_HOST} > /dev/null 2>&1
```

Comment:
Kubernetes/OpenShift will establish the connection only once the target service has a container passing the "readiness" check. We don't have such a check right now, but this is more like a hack and might eventually fail on the situation you just mentioned.

I think @jsanda can help us here in coming up with a reliable solution, but I'd prefer to have it as part of another task.